wger/gym/forms.py

```diff
@@ -33,13 +33,16 @@ class GymUserPermisssionForm(forms.ModelForm):
 
     # Empty default roles, they are always set at run time
     ROLES = ()
+    choices = (("deactivate","deactivate"), ("remove","remove"))
 
     class Meta:
         model = User
         fields = ('role',)
 
     role = forms.MultipleChoiceField(choices=ROLES,
                                      initial=USER)
+    # For when we need to change the UI
```

Comment:
Is this piece of code required for a later time? If it is notplease consider removing it to keep the code lean.